lfview/resources/spatial/options.py

```diff
@@ -113,7 +98,38 @@ def _validate_data(self):
             )
 
 
-class OptionsSize(OptionsOpacity):
+class OptionsTexture(_BaseOptionsItem):
+    """Options for a displayed texture"""
+
+    opacity = properties.Instance(
```

Comment:
Let's just keep this as `value = properties.Float(...)` for now. This keeps new db objects consistent with old.

This reverts:
- nested dictionary: we can introduce this extra complexity if and when it is needed for more complex texture mapping / data. For now, we can keep it simple.
- better named prop: it would be nice to have `opacity` rather than ambiguous `value` - but everything in the DB already has `value` so this change would require migration or property value manipulation (e.g. `Renamed` property). Might as well just wait to tackle this, possibly at the same time we move to a nested dictionary.